integration_testing/features/delete-resources.feature

```diff
@@ -0,0 +1,27 @@
+Feature: Delete resources permanently
+
+	Background: 
+		Given I am signed in to Studio as a non-admin user
+			And I am on the channel editor page
+			And I have one removed resource from my channel editor tree
+
+	Scenario: Delete a single resource permanently
+		When I click the *Trash* button in the top appbar
+		Then I see one resource removed from my channel
+		When I select that resource
+			And I click *Delete* in the bottom bar
+		Then I see a *Warning* message appear

```

Comment:
Replace all the instances of `*Warning*` with plain `warning` because we only use pairs of `*` to enclose the **actual** UI strings.